cfpb/consumerfinance.gov
# Remove export Ask data script

Now that we have a Wagtail report to export Ask CFPB data (see #7212), we can get rid of the old custom script that was used previously to do the same thing.

---

## Removals

- Ask export script

## How to test this PR

1. Log into the Wagtail admin
2. Confirm that the "Export Ask data" no longer appears at the bottom of the sidebar menu
3. Confirm the old management command that could be used to run that script from the command line, `python cfgov/manage.py runscript export_ask_data`, no longer works

## Checklist

  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing, e.g., "Mega Menu: fix layout bug", or "Docs: Update Docker installation instructions".
  - [This repo’s docs](https://cfpb.github.io/consumerfinance.gov/) (edit the files in the `/docs` folder) – for basic, close-to-the-code docs on working with this repo
  - CFGOV/platform wiki on GHE – for internal CFPB developer guidance
  - CFPB/hubcap wiki on GHE – for internal CFPB design and content guidance
